test/unittests/t_cache_ram.cc

```diff
@@ -23,7 +24,7 @@ namespace cache {
 class T_RamCacheManager : public ::testing::Test {
  public:
   T_RamCacheManager()
-    : ramcache_(4*alloc_size, 3*cache_size, &statistics_) {
+    : ramcache_(4*alloc_size, cache_size, &statistics_) {
```

Comment:
Since LRU doesn't have default or copy constructors, I had to do the initialization just so. I strongly suspect that gtest uses a new instance for each test, since I'm not particularly careful about freeing resources between tests. The `OpenClose` test runs out of fds exactly when expected for a fresh instance, and if I comment out the close loop at the end, the test behavior is the same.